crates/sui-framework/packages/sui-framework/sources/crypto/bls12381.move

```diff
@@ -171,6 +173,22 @@ public fun g1_multi_scalar_multiplication(
     group_ops::multi_scalar_multiplication(G1_TYPE, scalars, elements)
 }
 
+/// Convert an `Element<G1>` to uncompressed form.
+public fun g1_to_uncompressed(e: &Element<G1>): Element<G1Uncompressed> {
+    group_ops::from(G1_TYPE, G1_UNCOMPRESSED_TYPE, e)
+}
+
+/// Create a `Element<G1>` from its uncompressed form.
+public fun g1_from_uncompressed(e: &Element<G1Uncompressed>): Element<G1> {
```

Comment:
g1_uncompressed_to_g1 (basically we should be able to do `g1 = e.g1_uncompressed_to_g1()`)

Reply:
also, let's move the g1_uncompressed_* apis to its own "section" at the end (like we have for the other types)

Reply:
> Member

That's not possible, unfortunately,  because the type `Element` is defined in the `group_ops` module, and the `g1_uncompressed_to_g1` function is defined in the `bls_12381` module.